content/enterprise_influxdb/v1.9/administration/monitor/overview.md

```diff
@@ -0,0 +1,358 @@
+---
```

Comment:
I think a lot of the content in this doc was meant to land on the monitor `_index.md` page.